### PR TITLE
Added ICON R package to list

### PR DESCRIPTION
**Pull request by rrrlw**
Please complete the following checklist if your PR is adding new link to the list:

- [x] I've read [contributing guidelines](https://github.com/MunGell/awesome-for-beginners/blob/master/CONTRIBUTING.md)
- [x] This PR does not introduce duplicates to the list
- [x] The link is added at the bottom of the list category
- [x] Suggested repository is maintained, have active community, is able to mentor new contributors and have issues with the suggested label
- [x] The link I add follows the suggested pattern:

```
- [Repository Name](link-to-repository-label) _(label: beginner-friendly-label-in-the-repository)_ <br> Description
```

Example link formatting:

```
- [awesome-for-beginners](https://github.com/MunGell/awesome-for-beginners/labels/good-first-contribution) _(label: good-first-contribution)_ <br> A list of awesome beginners-friendly projects.
```

### CODE REVIEW

**Comment by rrrlw** [comment]
Anything I can do to avoid the issue automatically closing?

File: README.md [hunk 1]
```diff
@@ -229,7 +229,7 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 
 ## R
 
-- [ICON](https://github.com/rrrlw/ICON/issues/1) _(label: good first issue)_ <br> R data package providing cleanly formatted complex network datasets from the Index of COmplex Networks (ICON) database.
+- [ICON](https://github.com/rrrlw/ICON/labels/good%20first%20issue) _(label: good first issue)_ <br> R data package providing cleanly formatted complex network datasets from the Index of COmplex Networks (ICON) database.
```
Comment (prakharjn12) [inline README.md]: nice

Comment (rrrlw) [reply README.md]: Thank you!

**Comment by rrrlw** [comment]
Apologies, not sure what I clicked that changed the interface. Are there any changes I need to make prior to this pull request being merged?

**Comment by MunGell** [comment]
Hi @rrrlw 

Sorry for the delay in response.
It looks like your repository is very new, has only 1 contributor and doesn't have any open beginner-friendly issues at the moment.

I am closing this PR, but feel free to re-open it when your project will get a community around it strong enough to support new members!

**Pull request closed without merging**